### PR TITLE
Use bazelrc if available and create /tf directory if needed for local…

### PR DESCRIPTION
**Pull request by hsharsha**
…_xla tests

### CODE REVIEW

**Review by i-chaochen — APPROVED** [review]
Thanks!

**Review by i-chaochen — APPROVED** [review]

File: tensorflow/tools/ci_build/linux/rocm/run_xla.sh [hunk 1]
```diff
@@ -44,21 +44,44 @@ else
...
+		-k \
+		--test_tag_filters=-no_oss,-oss_excluded,-oss_serial,gpu,requires-gpu,-no_gpu,-no_rocm --keep_going \
+		--build_tag_filters=-no_oss,-oss_excluded,-oss_serial,gpu,requires-gpu,-no_gpu,-no_rocm \
+    		--config=rocm \
+    		--test_output=errors \
+    		--local_test_jobs=${N_TEST_JOBS} \
+    		--test_env=TF_TESTS_PER_GPU=$TF_TESTS_PER_GPU \
+    		--test_env=TF_GPU_COUNT=$TF_GPU_COUNT \
+    		--action_env=XLA_FLAGS=--xla_gpu_force_compilation_parallelism=16 \
+    		-- @local_xla//xla/... @local_xla//build_tools/...
```
Comment (i-chaochen) [inline tensorflow/tools/ci_build/linux/rocm/run_xla.sh]: I don't think we need to add `@local_xla//build_tools/...`

This is more NV's stuff in upstream. https://github.com/openxla/xla/tree/main/build_tools

**Review by i-chaochen — APPROVED** [review]

**Comment by hsharsha** [comment]
retest Ubuntu-GPU-multi please

**Comment by hsharsha** [comment]
retest Ubuntu-GPU-single please

**Pull request merged**